uvicorn/server.py

```diff
@@ -64,9 +65,7 @@ def __init__(self, config: Config) -> None:
 
     def run(self, sockets: Optional[List[socket.socket]] = None) -> None:
```

Comment:
The return is not None anymore, right?

Reply:
yes it's `-> None`, `asyncio.run(serve(...))` returns synchronously the `StopIteration.value` of the coro from `serve(...)`